tests/aws/services/events/test_events.py

```diff
@@ -817,7 +654,6 @@ def test_list_event_buses_with_limit(self, create_event_bus, aws_client, snapsho
     @markers.aws.unknown
```

Comment:
not yet validated?

Reply:
fixed in: https://github.com/localstack/localstack/pull/10664/commits/d2b4586aebd3486090d03bc97244f06873ed1eed